### DISCUSSION TITLE
How to identify which provider was used?

### DISCUSSION THREAD

**Discussion by mdavish** [nextauthjs/next-auth#3207] [Help]
Hi all - I am making an app that uses multiple providers, but I need to make some slight tweaks to logic depending on _which_ provider was used to log in. (I.e. did the user sign in with Google, GitHub, or Twitter?)

Is there an easy way to find this out?

**Comment by balazsorban44** [comment]
check `account` https://next-auth.js.org/configuration/callbacks#jwt-callback
